tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "extends": "aegir/src/config/tsconfig.aegir.json",
   "compilerOptions": {
-    "outDir": "dist",
-    "noImplicitAny": true
+    "outDir": "dist"
```

Comment:
I think we should keep `"noImplicitAny": true` where we can

```suggestion
    "outDir": "dist"
     "noImplicitAny": true
```

Reply:
this is already in the aegir ts config :)

Reply:
I don't see it here https://github.com/ipfs/aegir/blob/master/src/config/tsconfig.aegir.json

Reply:
comes default with `strict: true`

Reply:
Nor here https://github.com/ipfs/aegir/blob/feat/docs2.0/src/config/tsconfig.aegir.json which is what this depends on

Reply:
Got it, never mind then.